src/index.js

```diff
@@ -219,13 +219,42 @@ function isFingerprintMatch(fingerprint, device) {
     return match;
 }
 
+function findByModel(model){
+    /*
+    Search device description by model name.
+    Useful when redefining, expanding device descriptions in external converters.
+    */
+    const modelToFind = model.toLowerCase();
+    let candidate = findByZigbeeModel(modelToFind);
```

Comment:
Can't we just do `const definition = definitions.find((d) => d.model == model));` here?

Reply:
what if model in whitelabel ?

Reply:
ah, just in case, we are looking for zigbee model

Reply:
I think definition `model` is enough, zigbeeModel will just complicate stuff.

Reply:
ok. waiting